spec.go

```diff
@@ -225,6 +225,60 @@ type SourceBuild struct {
 	Args map[string]string `yaml:"args,omitempty" json:"args,omitempty"`
 }
 
+// SourceInlineContent is used to specify the content of an inline source.
+type SourceInlineFile struct {
+	// Content is the contents.
+	Contents string `yaml:"content,omitempty" json:"content,omitempty"`
+	// Permissions is the octal file permissions to set on the file.
+	Permissions fs.FileMode `yaml:"permissions,omitempty" json:"permissions,omitempty"`
+	// UID is the user ID to set on the directory and all files and directories within it.
+	// UID must be greater than 0
```

Comment:
It looks like in the validation code we're just testing `< 0`, could we clarify this comment vs. the validation logic?

Reply:
Fixed